config/routes.rb

```diff
@@ -400,6 +400,12 @@
         post '/complete' => 'safeguarding#complete', as: :complete_safeguarding
       end
 
+      scope '/feedback-form' do
+        get '/' => 'feedback_form#new', as: :feedback_form
+        post '/' => 'feedback_form#create', as: :create_feedback_form
```

Comment:
Minor suggestion - not super-important as it's not something we're consistent with in the routes, but we don't need the `create_feedback_form_path`. Can remove it and just POST to the `feeback_form_path` in the form for the same result.